src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -103,33 +99,46 @@ const TopicPage = ({ pageData }) => {
               title: curationTitle,
               link,
               position,
-              visualStyle,
-            }) => (
-              <React.Fragment key={`${curationId}-${position}`}>
-                {shouldRenderMessageBanner(visualStyle, visualProminence) ? (
-                  <MessageBanner
-                    heading={curationTitle}
-                    position={position}
-                    description={summaries[0].description}
-                    link={summaries[0].link}
-                    linkText={summaries[0].title}
-                    image={summaries[0].imageUrl}
-                  />
-                ) : (
-                  <Curation
-                    headingLevel={curationTitle && 3}
-                    visualStyle={visualStyle}
-                    visualProminance={visualProminence}
-                    promos={summaries}
-                    title={curationTitle}
-                    topStoriesTitle={topStoriesTitle}
-                    position={position}
-                    link={link}
-                    curationLength={curations && curations.length}
-                  />
-                )}
-              </React.Fragment>
-            ),
+              visualStyle = NONE,
+            }) => {
+              const component = getComponent(visualStyle, visualProminence);
+
+              return (
+                <React.Fragment key={`${curationId}-${position}`}>
+                  {(() => {
+                    switch (component) {
+                      case MESSAGE_BANNER:
+                        return (
+                          <MessageBanner
+                            heading={curationTitle}
+                            position={position}
+                            description={summaries[0].description}
+                            link={summaries[0].link}
+                            linkText={summaries[0].title}
+                            image={summaries[0].imageUrl}
+                          />
+                        );
+                      case BILLBOARD:
+                        return null;
```

Comment:
I added this because I wanted to remove https://github.com/bbc/simorgh/blob/latest/src/app/pages/TopicPage/Curation/index.jsx#L38 from the Curation object (and doing so caused one of the TopicPage tests to fail).

Reply:
This is related to https://github.com/bbc/simorgh/pull/10599#issuecomment-1428734001

Because I removed the check for a banner in the curation component, there's a risk that if editorial add a curation with visual style of banner, but a prominence of anything other than NORMAL or HIGH, then it will be rendered as a default Curation grid. 

Should we try to cater for these additional scenarios? Technically there are React errors/warnings in the console log when we visit our test topic page: because it's rendering components with visual style = FEED, LINKS etc, which are failing the prop-types rules (see https://github.com/bbc/simorgh/blob/latest/src/app/pages/TopicPage/Curation/index.jsx#L70-L71). 

Or should we fix this in a separate PR / new ticket which would have to be prioritised against all others?

Reply:
Doing it in this PR fits the purpose of this PR (according to the title), if we want to add a different way of dealing with Banner with different prominences. Happy to work on this and also look at the warnings, although the warnings could be done in a different PR if we want to get this merged sooner and don't care much about them

Reply:
Shall we just do it as part of this? Let's put the current permutations of visual style / prominence here, and then we can stack case statements in the TopicPage component to return null for components which we don't yet know how to render (which is probably only the banner for the moment - the others seem to work OK as the default CurationGrid)